### ISSUE TITLE
The -var option seems totally broken in Terraform 0.8

### ISSUE THREAD

**Issue by brikis98** [hashicorp/terraform#10716]
### Terraform Version

Terraform v0.8.0

### Affected Resource(s)

All

### Terraform Configuration Files

Very simple repro example:

```hcl
variable "foo" {}
```

### Expected Behavior

Here's what happens with Terraform 0.7.x:

```
> terraform apply -var 'foo = "bar"' -input=false

Apply complete! Resources: 0 added, 0 changed, 0 destroyed.
```

### Actual Behavior

Here's what happens with the exact same template in Terraform 0.8.0:

```
> terraform apply -var 'foo = "bar"' -input=false
There are warnings and/or errors related to your configuration. Please
fix these before continuing.

Errors:

  * 1 error(s) occurred:

* Required variable not set: foo
```

### Steps to Reproduce

Run `terraform apply` and pass variables using the `-var` option.

**Comment by mitchellh** [comment]
Ah, this is a regression but not "totally" broken: remove the space around the equal sign:

```
terraform apply -var 'foo="bar"`
```

Works. This is still a bug, but at least it mostly works!

**Comment by brikis98** [comment]
Ah, I believe you're right. I'll update our test code to get rid of the space for now. Thanks for the workaround.

**Comment by mitchellh** [comment]
Fix is in PR, will be fixed in 0.8.1

**Comment by brikis98** [comment]
Thx!
